test/clj/rems/api/test_api.clj

```diff
@@ -70,26 +72,25 @@
                                                                :path "/api/c.*"}
                                                               {:method "get"
                                                                :path "/api/users/.*"}]})
-    (testing "> api key without whitelist can access any path >"
-      (doseq [path ["/api/applications" "/api/my-applications" "/api/catalogue"]]
-        (testing path
-          (is (response-is-ok? (api-response :get path nil
-                                             "45" "owner"))))))
-    (testing "> api key with whitelist can access only given paths >"
+  
+  (testing "> api key without whitelist can access any path >"
+    (doseq [path ["/api/applications" "/api/my-applications"]]
+      (testing path
+        (is (response-is-ok? (api-response :get path nil
+                                          "45" "owner"))))))
+  (testing "> api key with whitelist can access only given paths >"
       (doseq [path ["/api/applications" "/api/my-applications"]]
         (testing path
           (is (response-is-ok? (api-response :get path nil
-                                             "46" "owner")))))
-      (is (response-is-unauthorized? (api-response :get "/api/catalogue" nil
-                                                   "46" "owner"))))
-    (testing "> api key with whitelist can access only matching paths >"
+                                              "46" "owner"))))))
+  (testing "> api key with whitelist can access only matching paths >"
```

Comment:
The identation looks wrong here but also shouldn't all these be changed to `/api/my-applications` and not just some?

Reply:
<img width="801" alt="Screenshot 2020-05-19 at 11 30 57" src="https://user-images.githubusercontent.com/7344718/82303923-4a745880-99c4-11ea-9673-8c213037d376.png">
It looks ok in my editor

Reply:
I guess @Macroz meant that there's still an `/api/catalogue` in the third test case, but that's fine by me as the test passes. It's kinda good to test a public api versus the whitelist too.

About indentation: note how the first `testing` has a two-space indent after it, (right before the `(doseq`, but the others have a four-space indent (right before `(doseq